shared/fs/common/security-prefs.js

```diff
@@ -0,0 +1,123 @@
+// @flow
+import React from 'react'
+import {BackButton, Box, Text} from '../../common-adapters'
+import {globalStyles, globalColors, platformStyles} from '../../styles'
+import {fileUIName} from '../../constants/platform'
+
+type Props = {
+  back: () => void,
+  openSecurityPrefs: () => void,
+  needAction: boolean,
+}
+
+const InstallSecurityPrefs = (props: Props) =>
+  props.needAction ? (
+    <Box style={stylesContainer}>
+      <BackButton key="back" onClick={props.back} style={stylesClose} />
+      <Text type="HeaderBig" style={{paddingBottom: 13, paddingTop: 10}}>
+        Ghhh. Try this.
+      </Text>
+      <Text type="Body" style={{paddingBottom: 24}}>
+        Open your macOS Security & Privacy Settings and follow these steps.
+      </Text>
+      <Box style={{...globalStyles.flexBoxRow, marginRight: 20}}>
+        <Box style={{position: 'relative'}}>
+          <img
```

Comment:
if this is desktop only it should be a .desktop file. otherwise we can't have raw html tags in here